src/ontology/cl-edit.owl

```diff
@@ -24989,6 +24989,9 @@ EquivalentClasses(obo:CL_1000217 ObjectIntersectionOf(obo:CL_0000138 ObjectSomeV
 
 # Class: obo:CL_1000222 (stomach neuroendocrine cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1196/annals.1294.001") obo:IAO_0000115 obo:CL_1000222 "A specialised neuroendocrine cell located in the gastric mucosa that regulates digestive processes including acid secretion and motility. This cell stores hormones in large dense core vesicles and synaptic-like microvesicles.")
```

Comment:
do you mean gut motility or cell motility? 
if it is cell motility include 'capable of' some 'cell motility' in the logical definition

Reply:
Gut motility, updated def to reflect this